plugins/offboard/offboard_impl.cpp

```diff
@@ -459,6 +459,8 @@ Offboard::Result OffboardImpl::offboard_result_from_command_result(MAVLinkComman
             return Offboard::Result::COMMAND_DENIED;
         case MAVLinkCommands::Result::TIMEOUT:
             return Offboard::Result::TIMEOUT;
+        case MAVLinkCommands::Result::NO_SETPOINT_SET:
```

Comment:
But `MAVLinkCommands::Result::NO_SETPOINT_SET` does not exist:
https://github.com/Dronecode/DronecodeSDK/blob/4332994c55c9ac8357954ccd1afcc5fa4a87172a/core/mavlink_commands.h#L19-L28

Reply:
OK, just added.